plugins/modules/ec2_vpc_endpoint.py

```diff
@@ -29,6 +29,20 @@
     choices: [ "Interface", "Gateway", "GatewayLoadBalancer" ]
     type: str
     version_added: 1.5.0
+  vpc_endpoint_subnets:
+    description:
+      - The list of subnets to attach to the endpoint (Works only with Interface and GatewayLoadBalancer type endpoint).
+    required: false
+    type: list
+    elements: str
+    version_added: 1.6.0
```

Comment:
```suggestion
    version_added: 2.0.0
```
There will not be a 1.6 of this collection, the next release will be 2.0.0.

Reply:
Thanks, I made the change